src/In.ProjectEKA.HipService/Link/LinkPatient.cs

```diff
@@ -32,11 +31,11 @@ public class LinkPatient : ILink
             this.discoveryRequestRepository = discoveryRequestRepository;
         }
 
-        public async Task<Tuple<PatientLinkReferenceResponse, ErrorResponse>> LinkPatients(
-            HipLibrary.Patient.Model.Request.PatientLinkReferenceRequest request)
+        public async Task<Tuple<PatientLinkEnquiryRepresentation, ErrorRepresentation>> LinkPatients(
+            PatientLinkEnquiry request)
         {
             var (patient, error) = PatientAndCareContextValidation(request);
-            if (error != null) return new Tuple<PatientLinkReferenceResponse, ErrorResponse>(null, error);
+            if (error != null) return new Tuple<PatientLinkEnquiryRepresentation, ErrorRepresentation>(null, error);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4942877)